io.github.shmmodbus.yml

```diff
@@ -0,0 +1,268 @@
+id: io.github.shmmodbus
```

Comment:
this app-id doesn't meet the requirements. It should have tbe user/org AND the application name. Like it's documented.

Reply:
i changed it to ```io.github.shmmodbus.shm-modbus```